BFWControls/Modules/Layer/View/UIView+Transform.swift

```diff
@@ -13,8 +13,8 @@ extension UIView {
     
     var rotationDegrees: CGFloat {
         get {
-            // TODO: Reverse engineer angle from transform.
-            return 0.0
```

Comment:
`CGFloat(M_PI)` is deprecated. Please use `.pi`, according tot eh Xcode 8.3/Swift 3.1 compiler, like you see in the `set`.